lib/text.rb

```diff
@@ -0,0 +1,12 @@
+class Text_message
```

Comment:
I think this is a great alternative to the Twilio API and provides the functionality on producing a message with the delivery time